applications/ad_diff/model_ad_diff.py

```diff
@@ -17,9 +17,6 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-import sys
```

Comment:
I'd like to keep hIPPYlib developer friendly, so `applications` should still work even if `hIPPYlib` is not PIP installed

Reply:
I will undo the changes in applications then